dotnet/src/webdriver/Logs.cs

```diff
@@ -74,21 +76,25 @@ public ReadOnlyCollection<string> AvailableLogTypes
         /// <param name="logKind">The log for which to retrieve the log entries.
         /// Log types can be found in the <see cref="LogType"/> class.</param>
         /// <returns>The list of <see cref="LogEntry"/> objects for the specified log.</returns>
+        /// <exception cref="ArgumentNullException">If <paramref name="logKind"/> is <see langword="null"/>.</exception>
         public ReadOnlyCollection<LogEntry> GetLog(string logKind)
         {
+            if (logKind is null)
+            {
+                throw new ArgumentNullException(nameof(logKind));
```

Comment:
The existing behavior for passing in null is exceptional 

![image](https://github.com/user-attachments/assets/c17a53ad-3f98-462f-8057-94a0dc95a95a)

Reply:
Thx, so null is not allowed.